lib/GenomeFileUtil/core/GenomeInterface.py

```diff
@@ -128,54 +193,75 @@ def get_one_genome(self, params):
         return data, res['info']
         # return self.dfu.get_objects(params)['data'][0]
 
-    def save_one_genome(self, params):
```

Comment:
Are there any changes in this deleted code other than moving it into a loop?

Reply:
I also moved the `ws_name_to_id` logic into _set_up_single_params.

Reply:
:+1: